docs/engineering/code-development.md

```diff
@@ -1,78 +1,100 @@
-# Backend Development
+# Code Development / Text Editors
 
 ## Table of Contents
 
-* [General Principles](#general-principles)
+* [Popular Editors at CivicActions](#popular-editors-at-civicactions)
+  * [Editor Configuration for Coding](#editor-configuration-for-coding)
+* [General Coding Principles](#general-coding-principles)
 * [Coding Standards](#coding-standards)
-  * [Drupal specific-coding](#coding-standards)
+  * [Drupal specific coding](#coding-standards)
 * [Debugging](#debugging)
 * [Git Commits](#git-commits)
-* [Editor Configuration](#editor-configuration)
 * [Code Examples](#code-examples)
   * [Don't use continue in short loops](#continue)
-  * [Simplify Boolean Logic](#boolean)
+  * [Simplify Boolean Logic](#simplify-boolean-logic)
+* [Still have questions?](#still-have-questions)
 
-## <a name="general-principles"></a>General Principles
+## Popular Editors at CivicActions
```

Comment:
I think this page should stay named "back-end development" - the page names should correspond to practice areas (the columns in our Trello board).